### DISCUSSION TITLE
关于globalData频繁修改和取值

### DISCUSSION THREAD

**Discussion by Wang-Junwen** [kbengine/kbengine#1374] [Q&A]
要做一个世界boss的玩法，想把boss的血量放在globalData里，这样的频繁修改和取值，会有性能影响吗？

如果这个血量是个自定义的python类，是不是类的内部属性发生变化时，会同步不到所有baseapp

**Comment by worldg** [comment]
你可以自己评估一下。 每次改动都会广播到所有baseapp和cellapp进程上。改动频率和数据大小
